docs/pipeline/datastreams.mdx

```diff
@@ -0,0 +1,103 @@
+---
+title: "Datastreams"
+description: "Datastreams let you split your observability data into multiple, independent pipelines - each routed to its own destination and optionally enhanced with custom actions like sampling, enrichment, or masking."
+sidebarTitle: "Datastreams"
+icon: "list"
+---
+
+import BetaWarning from '/snippets/beta-warning.mdx';
+
+<BetaWarning />
+
+# Introduction
+Odigos supports Datastreams, a powerful feature that allows you to split telemetry sources into multiple logical streams. Each stream can be independently routed to different destinations and have specific actions applied—such as sampling, attribute enrichment, masking rules, and more.
+
+# Motivation
+There are many use cases for adopting Datastreams, including:
+
+1. **Cost optimization** - Route lower-priority services (like internal or backoffice apps) to more cost-effective destinations, such as Parquet files stored in S3.
+
+2. **Targeted processing** - Apply custom processing (e.g., sampling, enrichment) to only a subset of services.
+
+3. **Gradual migration** - Test a new destination on specific services before committing to a full migration.
+
+4. **Organizational segmentation** - Split data based on development teams, product areas, or environments.
+
+# Creating Datastreams
+
+<Tabs>
+
+<Tab title="UI">
+After [installing Odigos CLI](/setup/installation), run the following command to start the Odigos UI:
+```bash
+odigos ui
+```
+In a **fresh Odigos installation**, you'll be taken to the Setup page. This is where your journey with Odigos begins, and you'll be asked to provide a few initial details:
+
+2. Provide a name for the Datastream
+3. Select the telemetry [Sources](/pipeline/sources/introduction) to include
+4. Choose the [destination(s)](/backends-overview) to send the data to
+5. Click **Done**
+
+If **Odigos is already installed**, you can create a new Datastream from the Overview page. Click the `+ NEW` button at the top-left (under the current Datastream views) and follow the same steps as in the setup phase.
```

Comment:
maybe this tab could have 2 smaller headers: `In a new Odigos install` and `In an existing Odigos install` or something like that